how-to-guides.rst

```diff
@@ -16,10 +16,10 @@ In the Netmaker UI, that node is set as an "egress gateway." Range(s) are specif
 
    egress-gateway
 
-Ingress Gateway/External Clients
+Remote Access Gateway/External Clients
```

Comment:
increase the underline. sometimes the titles dont work properly when you dont fully underline

Reply:
done